spec/requests/stories_index_spec.rb

```diff
@@ -2,17 +2,23 @@
 
 RSpec.describe "StoriesIndex", type: :request do
   describe "GET stories index" do
-    it "renders page with proper sidebar" do
+    let!(:article) { create(:article, featured: true) }
+
+    it "renders page with article list" do
       get "/"
-      expect(response.body).to include("key links")
+      expect(response.body).to include(article.title)
     end
 
     it "renders page with min read" do
-      create(:article, featured: true)
       get "/"
       expect(response.body).to include("min read")
     end
 
+    it "renders page with proper sidebar" do
```

Comment:
I'm not sure how valuable this particular test is as anywhere in the body could include the words `'key links'` no?

Reply:
Fair point! What's odd is that "Key Links" widget it's testing doesn't have an id at all (and frankly: testing template output in a request test is very weird). I've changed it to look for the `h4` tag specifically.